src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
 // write your code here
+const getLogo = document.querySelector('.logo');
+const createDiv = document.createElement('div');
+
+createDiv.classList.add('message');
+
+const promise1 = new Promise((resolve) => {
+  getLogo.addEventListener('click', () => {
+    if (resolve) {
```

Comment:
The condition `if (resolve)` is unnecessary because `resolve` is always a function provided by the Promise constructor. You can directly call `resolve('Promise was resolved!')`.